fs.js

```diff
@@ -1,1705 +0,0 @@
-// Copyright Joyent, Inc. and other Node contributors.
```

Comment:
from what I can tell this is an (extremely) out of date version of the node fs library rewritten to use readable streams, unclear if it would even work as it hasn't been updated in a long time